design.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="style_design.css">
+</head>
+<body>
+    <div class="wrap">
+        <header class="header">
+           <div class="logo">
+            <a class="modus" href="#">MODUS<span class="versus">versus</span></a>
+           </div>  
+            <div class="nav">
+             <nav class="nav-list">
+                <ul>
+                    <li class="item">
+                        <a href="#">HOME</a>
+                    </li>
+                    <li class="item">
+                        <a href="#">ABOUT</a>
+                    </li>
+                    <li class="item">
+                        <a href="#">SERVICES</a>
+                    </li>
+                    <li class="menu">
+                        <a  data-toggle="dropdown" href="#"><span class="caret">PORTFOLIO</span></a>
+                         <ul class="sub-menu">
+                             <li><a href="#"><img class="image" src="dropdown-menu_files/dropdown_linkbg.gif"></a></li>
```

Comment:
name class for image should be detail, ex: `image-arrow`, currently, `image` class  not use for all images